### ISSUE TITLE
Error! Server failure or wrong URL.

### ISSUE THREAD

**Issue by ddgweb** [MinhasKamal/DownGit#26]
https://github.com/aspnetboilerplate/aspnetboilerplate-samples/tree/master/SimpleTaskSystem-Core
Create Download LinkDownload

**Comment by Metroite** [comment]
I'm having the same problem. Can confirm. I really rely on this site! :S

**Comment by ifyour** [comment]
Same problem here.



> API rate limit exceeded for [YOUR IP]. (But here's the good news: Authenticated requests get a higher rate limit. Check out the [documentation](https://developer.github.com/v3/#rate-limiting) for more details.)

I need to add an authorization to get more request restrictions.🤔How to configure personal authorization?

**Comment by ddgweb** [comment]
> Same problem here.
> 
> > API rate limit exceeded for [YOUR IP]. (But here's the good news: Authenticated requests get a higher rate limit. Check out the [documentation](https://developer.github.com/v3/#rate-limiting) for more details.)

I know your mean, but when I send Download Link Request in your website, the request IP is your cloud server that your server's IP is the realy request github , no my pc local IP addres.

**Comment by eebssk1** [comment]
> 
> 
> > Same problem here.
> > > API rate limit exceeded for [YOUR IP]. (But here's the good news: Authenticated requests get a higher rate limit. Check out the [documentation](https://developer.github.com/v3/#rate-limiting) for more details.)
> 
> I know your mean, but when I send Download Link Request in your website, the request IP is your cloud server that your server's IP is the realy request github , no my pc local IP addres.

Nope,you are actually downloading with your own browser.there's no server that download the files for you.Just Your Browser.
You can check the browser console (network request)for this.

**Comment by thomastthai** [comment]
Just experienced this same error today. I'm logged in and don't use Github API much at all.

![image](https://user-images.githubusercontent.com/16532581/214615194-c6fd078b-e789-422c-8450-d9769a48b809.png)
